src/ExperimentUtils.py

```diff
@@ -0,0 +1,126 @@
+# File for making experiment functions.
```

Comment:
Note: Filenames should be `lower_case_with_underscores.py` by python convention --> Please use this convention for all your files